### PR TITLE
Yo Office Updates

### PR DESCRIPTION
**Pull request by nokafor**
Updated documentation to reflect changes to Yo Office flow. Changes will be released 6/22.

### CODE REVIEW

**Comment by jasonjoh** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/addin-tutorial.md](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/addin-tutorial.md) | :white_check_mark:Succeeded | |
[docs/add-ins/images/addin-tutorial/yeoman-prompts.PNG](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/images/addin-tutorial/yeoman-prompts.PNG) | :white_check_mark:Succeeded | |
[docs/add-ins/images/quick-start-yo-prompts.PNG](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/images/quick-start-yo-prompts.PNG) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201806221413417934-269%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lrFLjpQXqzSMDk9RECPMD2Cya7ZTl%2BtvO4hsAOgjHi8%3D&st=2018-06-22T14%3A09%3A34Z&se=2018-07-23T14%3A14%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/addin-tutorial.md [hunk 1]
```diff
@@ -35,18 +35,14 @@ You will also need Outlook 2016 connected to an Office 365 account, Outlook.com
 
 Open your command prompt/shell in an empty directory. Enter `yo office` and follow the prompts.
 
-- **Would you like to create a new subfolder for your project?:** `No`
-- **What would you want to name your add-in?:** `Git the Gist`
+- **Choose a project type:** `Jquery`
+- **Choose a script type:** `Javascript`
+- **What do you want to name your add-in?:** `My Office Add-in`
```
Comment (ElizabethSamuel-MSFT) [inline docs/add-ins/addin-tutorial.md]: Could you keep the name Git the Gist here?

Comment (nokafor) [reply docs/add-ins/addin-tutorial.md]: Good catch! Updated.

**Comment by jasonjoh** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/addin-tutorial.md](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/addin-tutorial.md) | :white_check_mark:Succeeded | |
[docs/add-ins/images/addin-tutorial/yeoman-prompts.PNG](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/images/addin-tutorial/yeoman-prompts.PNG) | :white_check_mark:Succeeded | |
[docs/add-ins/images/quick-start-yo-prompts.PNG](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/images/quick-start-yo-prompts.PNG) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/nokafor/outlook-dev-docs/blob/patch-1/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201806221544213213-269%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sbaD8d7mtENniVT%2FOTFrJFChZ%2FjbakP03TLymjo14WE%3D&st=2018-06-22T15%3A40%3A14Z&se=2018-07-23T15%3A45%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**